app_flutter/lib/widgets/task_grid.dart

```diff
@@ -198,45 +198,43 @@ class _TaskGridState extends State<TaskGrid> {
     int commitCount = 0;
     for (final CommitStatus status in filteredStatuses) {
       commitCount += 1;
-      for (final Stage stage in status.stages) {
```

Comment:
Here's an example of the general pay off we get from removing stages. We don't need 3 loops to scan the grid